js/components/utils.js

```diff
@@ -529,6 +529,21 @@ var Utils = {
     return (src && (typeof src === 'string' || src instanceof String))
   },
 
+  /**
+   * Returns sanitized config data
+   *
+   * @function sanitizeConfigData
+   * @param {Object} data to be sanitized
+   * @returns {Object} data if data is valid, empty object if not
+   */
+  sanitizeConfigData: function(data) {
```

Comment:
we don't accept playerParam.skin.inline being an `[]`

an Object with array is still acceptable `{"buttons": [1, 2, 3]}`

`sanitizeData` is generic `sanitizeConfigData` is just for CONFIG data